data/network/build.gradle.kts

```diff
@@ -53,64 +23,44 @@ kotlin {
             }
         }
 
-
         // android
-        getByName("androidMain") {
-            dependsOn(commonMain)
+        val androidMain by getting {
             dependencies {
                 implementation(libs.io.ktor.client.cio)
             }
         }
 
-
         // desktop
-        getByName("desktopMain") {
+        val desktopMain by getting {
```

Comment:
Changed access everywhere to the `by getting` syntax to be consistent.

Reply:
Yes thanks for doing this. It was bugging me that we were using different conventions across all the build files. Although I prefer named() or getByName() but I don't feel strongly. At least now it's consistent

Reply:
Don't feel strongly either, but the one benifit for the above syntax is you can't typo it doing:

```
val iosMain by creating {
     dependsOn(commonMain)
}
```

Reply:
Yup that's fine. As long as we stick with one moving forward than works for me 👍